modules/apps/object/object-admin-rest-test/src/testIntegration/java/com/liferay/object/admin/rest/resource/v1_0/test/ObjectDefinitionResourceTest.java

```diff
@@ -1026,6 +1028,56 @@ public void testPutObjectDefinition() throws Exception {
 		_objectDefinitionLocalService.deleteObjectDefinition(
 			randomModifiableSystemObjectDefinition.getId());
 
+		// Modifiable system object definition different language ID
```

Comment:
Can you rename this comment to `// Modifiable system object definition with a different language ID`